db/seeds.rb

```diff
@@ -15,6 +16,20 @@ def create_categories
     end
   end
 
+  def create_users
+    User.create!([
+      {
+        first_name: "Brant",
+        last_name: "Wellman",
+        address: "1510 Blake St",
+        city: "Denver",
+        zipcode: "80202",
+        email: "brant@email.com",
+        password: "password"
+      }
+      ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.